hnn_core/tests/test_params.py

```diff
@@ -13,3 +14,33 @@ def test_params():
     params = Params(params_fname)
     print(params)
     print(params['L2Pyr*'])
+
+
+def test_legacy_params():
+    """Test legacy params object."""
+    hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+    param_url = ('https://raw.githubusercontent.com/hnnsolver/'
+                 'hnn-core/test_data/default.param')
+    params_legacy_fname = op.join(hnn_core_root, 'param', 'default.param')
+    if not op.exists(params_legacy_fname):
+        _fetch_file(param_url, params_legacy_fname)
+
+    params_new_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params_legacy = Params(params_legacy_fname)
+    params_new = Params(params_new_fname)
+
+    if not params_new == params_legacy:
```

Comment:
maybe we can make all this a one liner by using: https://github.com/mne-tools/mne-python/blob/da5ffc58556c475db36b8c0f6bcf9df5df9a9519/mne/utils/numerics.py#L734 ?

Reply:
I see the benefit, but it seems unnecessary to pull in mne for this task. Line 32 above will do the job if placed in an assert statement. The print statements were used for my own purposes. I guess the output wouldn't even make it to the user.

Reply:
If you run pytest, you don't even see the outputs. In test driven development mode, you will run something like:

```sh
$ pytest hnn_core --pdb
```
and that drops you into pdb the moment you have an error